src/templates/static/legal/payment_agents.jsx

```diff
@@ -2,185 +2,182 @@ import React from 'react';
+            <li>{it.L('The Agent represents and warrants that the Agent is and will be in compliance with all laws related to anti-money laundering as well as any financial or economic sanction programmes in the jurisdiction in which he operates.')}</li>
+            <li>{it.L('Where the Agent omits, fails, or refuses in any manner to provide the Company with evidence of his identity or any other evidence as required by the law in relation to anti-money laundering within a reasonable time, the Company reserves the right to cease to deal with him.')}</li>
         </ol>
 
-        <h2 data-anchor='anti-corruption'>I. {it.L('Anti-Corruption')}</h2>
+        <h2 data-anchor='anti-corruption'>I. {it.L('Anti-corruption policy')}</h2>
         <ol>
-            <li>{it.L('The Agent undertakes to the Company that it shall comply with, and the service will be performed in accordance with any applicable anti-corruption laws.')}</li>
-            <li>{it.L('This obligation encompasses the prohibition of unlawful payments or the granting of any form of unlawful benefits to public officials, business partners, employees, family members or any other partners.')}</li>
-            <li>{it.L('The Company shall have the right to suspend or terminate this Agreement with immediete effect, should the Company have reasonable suspicion that the Agent is undertaking any practices that in breach of any applicable anti-corruption laws.')}</li>
+            <li>{it.L('The Agent promises to the Company that the Agent shall comply with, and the Agent\'s services will be performed in accordance with, any applicable anti-corruption laws.')}</li>
+            <li>{it.L('This obligation encompasses the prohibition of unlawful payments or the granting of any form of unlawful benefits to public officials, business partners, employees, family members, or any other partners.')}</li>
+            <li>{it.L('The Company shall have the right to suspend or terminate this agreement with immediate effect if the Company has reasonable suspicion that the Agent is undertaking any practices that are in breach of any applicable anti-corruption laws.')}</li>
         </ol>
-        <h2 data-anchor='onboarding-policy'>J. {it.L('On-boarding Policy')}</h2>
+        <h2 data-anchor='onboarding-policy'>J. {it.L('On-boarding policy')}</h2>
         <ol>
             <li>{it.L('The Agent shall submit an application including the following information:')}
                 <ul className='bullet'>
-                    <li>{it.L('name, email address and contact number;')}</li>
-                    <li>{it.L('URL website (where applicable);')}</li>
-                    <li>{it.L('a list of payment methods accepted;')}</li>
-                    <li>{it.L('the commission to be charged on deposits and withdrawals;')}</li>
-                    <li>{it.L('any other information as requested by the Company.')}</li>
+                    <li>{it.L('Name, email address and contact number')}</li>
```

Comment:
@AzitaRassi can it be "Name, email address, and contact number"?

Reply:
Indeed it is. A comma should be inserted before "contact number". Thank you.